packages/alias/test/test.js

```diff
@@ -333,12 +333,11 @@ test('Alias + rollup-plugin-node-resolve', (t) =>
     .then(getModuleIdsFromBundle)
     .then((moduleIds) => {
       const normalizedIds = moduleIds.map((id) => path.resolve(id)).sort();
-      t.is(normalizedIds.length, 7);
+      t.is(normalizedIds.length, 6);
```

Comment:
The missing file only contains a reexport and will no longer show up in the output as associated with any chunk.